recipes/_node.rb

```
@@ -1,13 +1,29 @@
-include_recipe 'nodejs::default'
-include_recipe 'nodejs::npm'
+# install node
+bash 'install node' do
+  code <<-EOC
+    curl -sL https://deb.nodesource.com/setup_#{node['node']['major_version']}.x | sudo -E bash -
```

I sure hope they don't change this naming convention. Is there something we can put in place to check for a successful response from the `curl` call?

---

It looks like they've been using it since version 6 and have scripts up to version 11 right now.  If they change it for some future version it would break this step and the cookbook would fail loudly (which is what I want)